packages/core-v2-sdk/src/utils/token/decodeTokenId.ts

```diff
@@ -0,0 +1,35 @@
+import { UNLOCKED_YT_ID } from "src/utils/constants";
+
+/**
+ * Get the data stored in a token id
+ * @param {string} tokenId - The token id to decode
+ * @return An object containing the data stored in the id
+ */
+export function decodeTokenId(tokenId: string): {
```

Comment:
ditto on having this function use seconds instead of ms. it will simplify the logic  a bit